### PR TITLE
Document the status property a bit more clearly.

### PR DESCRIPTION
**Pull request by WPettersson**
Details of what each property means are taken from lines ~144-146 of app-indicator.h (search for @APP_INDICATOR_STATUS_PASSIVE).

### CODE REVIEW

**Comment by WPettersson** [comment]
Just to clarify my position, I wanted to use the attention_icon in my code, and while I could work out that I needed to set a status to use it, the generated documentation was not at all clear on what values where valid (see e.g. https://lazka.github.io/pgi-docs/AyatanaAppIndicator3-0.1/classes/Indicator.html#AyatanaAppIndicator3.Indicator.props.status). The description might already be too wordy, but at the same time it's not immediately clear from the documentation what the "type" of the status should be. I ended up working out that I had to use IndicatorStatus.ATTENTION so maybe that's also worth having in the documentation.

As an aside, is there an "official" online documentation repo?

**Review by sunweaver — APPROVED** [review]
LGTM. Thanks for your contribution.
